src/Microsoft.ML.Transforms/OptionalColumnTransform.cs

```diff
@@ -26,6 +26,10 @@
 
 namespace Microsoft.ML.Runtime.DataPipe
 {
+    /// <summary>
+    /// This transform is used to mark some of the columns (e.g. Label) optional during training So that the columns is not required during scoring..
+    /// At scoring time, it is checked if the data schema for scoring matches the data used for training except for the optional columns.
```

Comment:
>At scoring time, it is checked if the data schema for scoring matches the data used for training except for the optional columns. [](start = 8, length = 129)

I'm not sure I actually understand this. What it really does is, if the column is *not* there, it will make it appear as if it is there with metadata filled in, and if the column is requested will return entirely default values. Or so I understand the code in the cursor. Is there some way to describe it in a way that communicates that? I'm not sure this sentence here actually communicates that.